File: go-sdk/pkg/models/operations/deleteuserv2.go
package operations

import (
	"openapi/pkg/models/shared"
)

type DeleteUserV2PathParams struct {
	ID string `pathParam:"style=simple,explode=false,name=id"`
}

type DeleteUserV2Request struct {
	PathParams DeleteUserV2PathParams
}

type DeleteUserV2Response struct {
	ContentType string
	StatusCode  int64
	User        *shared.User
}

File: user-api/ts-sdk/src/sdk/models/operations/deleteuserv2.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import * as shared from "../shared";



export class DeleteUserV2PathParams extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=id" })
  id: string;
}


export class DeleteUserV2Request extends SpeakeasyBase {
  @SpeakeasyMetadata()
  pathParams: DeleteUserV2PathParams;
}


export class DeleteUserV2Response extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;

  @SpeakeasyMetadata()
  user?: shared.User;
}
